owtf/db/database.py

```diff
@@ -43,11 +44,16 @@ def wrapper(self, *args, **kwargs):
 
 
 def get_db_engine():
-    engine = create_engine(
-        "postgresql+psycopg2://{}:{}@{}/{}".format(DATABASE_USER, DATABASE_PASS, DATABASE_IP, DATABASE_NAME),
-        pool_recycle=120)
-    Base.metadata.create_all(engine)
-    return engine
+    try:
+        engine = create_engine(
+            "postgresql+psycopg2://{}:{}@{}:{}/{}".format(DATABASE_USER, DATABASE_PASS, DATABASE_IP, DATABASE_PORT,
+                                                          DATABASE_NAME),
+            pool_recycle=120)
+        Base.metadata.create_all(engine)
+        return engine
+    except exc.OperationalError as e:
+        logging.error("Could not create database engine - Exception occured\n{}".format(str(e)))
```

Comment:
Make this `logging.error("Could not create database engine - Exception occured\n%s", str(e))` according to https://www.python.org/dev/peps/pep-0282/ best practices (some perf benefits).